draft-irtf-cfrg-hpke.md

```diff
@@ -340,8 +345,8 @@ def LabeledExpand(PRK, label, info, L):
 Suppose we are given a KDF, and a Diffie-Hellman group providing the
 following operations:
 
-- GenerateKeyPair(): Generate an ephemeral key pair `(sk, pk)`
-  for the DH group in use
+- DeriveKeyPair(ikm): Derive a key pair `(sk, pk)` from a byte string `ikm`
+  with at least `Nsk` bytes of entropy
```

Comment:
Does this mean that `ikm` has at least `Nsk` bytes of entropy? (That would seem to contradict the need for extract, I think.)

Reply:
(Chatted offline, to resolve with a slight tweak of text)